internal/observability/tracing/conf.go

```diff
@@ -77,9 +79,14 @@ func (c *Conf) Validate() error {
 		if c.OTLP.CollectorEndpoint == "" {
 			return errOTLPEndpointUndefined
 		}
+
 		return nil
 
 	default:
 		return fmt.Errorf("unknown trace exporter %s", c.Exporter)
 	}
 }
+
+func (c *Conf) SetDefaults() {
+	c.OTLP.Protocol = "grpc"
```

Comment:
I think this actually won't work because `c.OTLP` will be nil at this point.

Reply:
yep that's right I miss it